Main/DateColumn.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017. All rights reserved.
+ * Distributed under the terms of the MIT license.
+ *
+ * Authors:
+ *		Janus
+ */
+
+/*
+	Based on Tracker Modified Column
+*/
+
+#include <DateFormat.h>
+#include <DateTimeFormat.h>
+#include "DateColumn.h"
+
+// #pragma mark - DateColumn
+
+
+#define kTEXT_MARGIN	8
+
+
+DateColumn::DateColumn	(const char* title,
+								 float width,
+								 float minWidth,
+								 float maxWidth,
+								 alignment align ):
```

Comment:
Not sure about the large indention, and the 1 parameter/line, but I don't mind much...
What is amiss, is the space in front of the bracket: "alignment align ):"